tests/widget/modal/widget-modal.login-with-misc-callbacks.test.js

```diff
@@ -21,15 +22,21 @@ test('Modal widget with simple login and misc callbacks', async ({ page }) => {
   await expect(page.getByText('Are you human?')).toBeVisible();
   await clickButton('Yes', '/authenticate'); // <- Self submitting callback
 
-  // Choice
+  // Choice (select)
   const selectEl = page.getByLabel('Are you sure?');
   await selectEl.selectOption('0');
 
-  // Polling Wait
+  // Choice (radio)
+  const radioEl = page.getByLabel('Are you sure?');
+  await radioEl.selectOption('0');
+  await clickButton('Next', '/authenticate');
```

Comment:
Expanded the misc-callbacks test (and corresponding tree) to include choice as radio as well as select, now that we have a way to trigger it through the stage parameter.